ci-operator/jobs/infra-periodics.yaml

```diff
@@ -1006,37 +1006,6 @@ periodics:
     - name: token
       secret:
         secretName: github-credentials-openshift-merge-robot
-- agent: kubernetes
-  cluster: api.ci
-  decorate: true
-  decoration_config:
-    oauth_token_secret:
-      key: oauth
-      name: github-credentials-openshift-ci-robot-private-git-cloner
-  extra_refs:
-  - base_ref: master
-    org: openshift
-    repo: config
-    workdir: true
-  interval: 12h
-  labels:
-    ci.openshift.io/role: infra
-  name: periodic-openshift-priv-group-update
-  spec:
-    containers:
-    - args:
-      - --peribolos-config=org/config.yaml
-      - --org=openshift-priv
-      - --group=openshift-priv-admins
-      command:
-      - group-auto-updater
-      image: group-auto-updater:latest
-      imagePullPolicy: Always
-      name: ""
-      resources:
-        requests:
-          cpu: 500m
-    serviceAccountName: group-auto-updater
```

Comment:
@alvaroaleman the reason why we had the periodic is reconciling to the desired state when 1) postsubmit fails for any reason (could be limited GH outage or a network flake so that you do not rely on next merge, 2) when someone messes with the config manually for any weird reason. So the postsubmit gives you a chance to for quick commit, and the periodic keeps things consistent.

We have this pattern even for other things (boostrapper, applyconfig, even peribolos runs...), I don't think there's a race - at worst, two tools will race to update to the same thing - read real group membership and update to that. At worst the next periodic will reconcile any discrepancies. I think we should keep it.

Reply:
Yeah generally the periodic makes sense, it just doesn't make sense if it uses a repo in this config but we want a config based on the other repo

Reply:
That's AFAIK what the periodic does - it only reads peribolos config from openshift/config, not from here. We got rid of the race by removing the `Group` manifest that was present in this PR earlier.